harmonica/_transformations.py

```diff
@@ -393,6 +393,68 @@ def total_gradient_amplitude(grid):
     return np.sqrt(gradient[0] ** 2 + gradient[1] ** 2 + gradient[2] ** 2)
 
 
+def tilt_angle(grid):
+    r"""
+    Calculates the tilt angle of a potential field grid
+
+    Compute the tilt of a regular gridded potential field
+    :math:`M`. The horizontal derivatives are calculated
+    though finite-differences while the upward derivative
```

Comment:
Minor typo fix here.
```suggestion
    through finite-differences while the upward derivative
```